tests/Issue3278.py

```diff
@@ -0,0 +1,37 @@
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2025 Zachary Haberman <6599715+interifter@users.noreply.github.com>#
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+from . import Framework
+
+
+class Organization(Framework.TestCase):
```

Comment:
I think the change in the existing test is sufficient: https://github.com/PyGithub/PyGithub/pull/3291/files#diff-a9967421dab2a455d4a672a0b71fa7a1ce592788b6e8c85a565a119bda9474bbR217